FirebaseAuth/Tests/Unit/SwiftAPI.swift

```diff
@@ -31,7 +31,7 @@ class AuthAPI_hOnlyTests: XCTestCase {
   // Each function corresponds with a public header.
   func FIRActionCodeSettings_h() {
     let codeSettings = FirebaseAuth.ActionCodeSettings()
-    codeSettings.setIOSBundleID("abc")
+    codeSettings.iOSBundleID = "abc"
```

Comment:
Looks like a bug in the existing API, but we'll still need to call this out for API review.